src/symbols/Node.tsx

```diff
@@ -275,7 +275,7 @@ export const Node: FC<NodeProps> = ({
               color={color}
               node={node}
               active={combinedActiveState}
-              selected={isSelected || isActive}
+              selected={isSelected}
```

Comment:
You could remove this `selected` prop entirely, `Icon` doesn't use it

Reply:
@evgenoid ^^

Reply:
Yes, but it extends `NodeRendererProps` where `selected` is required property